create-waku-app/createApp.js

```diff
@@ -20,13 +23,28 @@ const init = (name, description, version, supportedExamples) => {
         })
         .option(
             "-t, --template <path-to-template>", 
-            "specify a template for the created project"
+            "specify a template for the created project or you can skip and select template manually"
```

Comment:
Change to "Or you can use the wizard" as selecting in the wizard is still "manual"